ir-engine/ir-engine
# IR-3575 playmode on scene change better fix

## Summary
playModeTool and sceneFunctions now react or use reactive state setting for isEditing state

## References
[https://tsu.atlassian.net/browse/IR-3575](https://tsu.atlassian.net/browse/IR-3575)

## QA Steps
enter a scene, enter playmode (with avatar) then load another scene

you should load into the new scene without issue, no longer in play mode and with the ability to start play mode again
